src/library.py

```diff
@@ -153,10 +153,15 @@ def _split_multigame_key(key: Key) -> List[KeyGame]:
         """Extract list of KeyGame objects from single Key"""
         logger.info(f'Spliting {key}')
         names = key.human_name.split(', ')
-        return [
-            KeyGame(key, f'{key.machine_name}_{i}', name)
-            for i, name in enumerate(names)
-        ]
+        games = []
+
+        for i, name in enumerate(names):
+            # Multi-game packs have the word "and" in front of the last game name
+            first, _, rest = name.partition(" ")
+            if first == "and": name = rest or first
+            logger.debug(f"Split game: {name}")
```

Comment:
I think we can remove this log. The final game name is logged in a different place already